custom.css

```diff
@@ -0,0 +1,18 @@
+.info {
+    margin: 20px;
+    padding: 0 20px;
+    border-inline-start: 2px solid var(--links);
+}
+
+.info:before {
+    position: absolute;
+    width: 3rem;
+    height: 3rem;
+    margin-inline-start: calc(-1.5rem - 21px);
+    content: "ⓘ";
+    text-align: center;
+    background-color: var(--bg);
+    color: var(--links);
+    font-weight: bold;
+    font-size: 2rem;
+}
```

Comment:
Would be nice to add no-copy prompts like in the Nix manual and Nixpkgs: https://github.com/NixOS/nixpkgs/pull/301439/files

Reply:
Nice! Done.